app/assets/javascripts/angular/directives/query/query_details_directive.js.es6

```diff
@@ -35,6 +36,18 @@
       this.resultRunner.run().then(this._tabs.navigateToTab.bind(this._tabs, 'results'));
     }
 
+    cloneQuery() {
+      let clone = angular.copy(this.query);
+      clone._item.id = undefined;
```

Comment:
lets encapsulate this logic in the `models` lib as a service like we talked about